config/routes.rb

```diff
@@ -1,8 +1,9 @@
 Myflix::Application.routes.draw do
-  root to: "pages#front"
+  root to: 'pages#front'
 
-  get 'home', to: "videos#index"
+  get 'home', to: 'videos#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.